scss/jquery/overrides/_select2.scss

```diff
@@ -0,0 +1,246 @@
+// Single select2
+.select2-container {
+  // The input field overlay
+  .select2-choice {
+    background: $crm-white;
+    border-color: $crm-grayblue-darker;
+    border-radius: $border-radius-base;
+    box-shadow: $crm-form-control-shadow;
+    box-sizing: border-box;
+    height: $input-height-base;
+    line-height: 27px;
+
+    &:focus {
+      outline: 0;
+    }
+
+    // The X icon to deselect the option
+    abbr {
+      background-image: none !important;
+      right: 30px;
+      top: 1px;
+
+      &::before {
+        @include fa-icon();
+        color: $brand-primary;
+        content: $fa-var-times-circle;
+        font-size: $font-size-base;
+      }
+    }
+
+    .select2-arrow {
+      background: $crm-white;
+      border-left-color: $crm-grayblue-darker;
+      border-radius: 0 $border-radius-base $border-radius-base 0;
+      text-align: center;
+      width: 26px;
+
+      &::before {
+        @include fa-icon();
+        color: $gray-darker;
+        content: $fa-var-caret-down;
+        font-size: $font-size-base;
+        position: relative;
+        top: -1px;
+        vertical-align: middle;
+      }
+
+      // The original arrow icon
+      b { /* stylelint-disable-line selector-max-compound-selectors */
+        display: none;
+      }
+    }
+
+    // The selected option
+    > .select2-chosen {
+      font-size: $font-size-base;
+    }
+  }
+}
+
+// Multiple select2
+.select2-container-multi {
+  // Use the magnifying glass icon when fetching data via ajax
+  &.crm-ajax-select {
+    .select2-choices {
+      &::before {
+        content: $fa-var-search;
+        right: 0;
+        top: 0;
+      }
+    }
+  }
+
+  // Wrapper
+  .select2-choices {
+    background: $crm-white;
+    border-color: $crm-grayblue-darker;
+    border-radius: $border-radius-base;
+    box-shadow: $crm-form-control-shadow;
+    line-height: 27px;
+
+    // By default use a caret down icon, without borders
+    &::before {
+      @include fa-icon();
+      background: none;
+      color: $gray-darker;
+      content: $fa-var-caret-down;
+      font-size: $font-size-base;
+      right: -3px;
+      top: 1px;
+    }
+
+    // The selected option
+    .select2-search-choice {
+      background: none;
+      border-color: $brand-primary;
+      border-radius: $border-radius-base;
+      font-size: $font-size-base;
+      padding: 3px 25px 3px 5px;
+
+      &.select2-search-choice-focus {
+        background: $gray-light;
+      }
+
+      /* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+      .select2-search-choice-close {
+        background: none !important;
+        left: auto;
+        right: 4px;
+        top: 4px;
```

Comment:
Same as above.

Reply:
Adjustment of the civicrm's style, same as above